typescript/src/cli.ts

```diff
@@ -31,6 +30,5 @@ async function main() {
 }
 
 main().catch(err => {
-  console.error(err);
-  process.exit(1);
+  throw new Error(err);
```

Comment:
Is there any value to doing this?  @bcoe   Honestly asking 😆

Reply:
That's the most confusing linter rule I ever saw :) I don't know why they hate `process.exit(1)` - do they care about non-Node environments?

Reply:
then can we just console.error some the err messages, instead of throwing another error? The linter hates `process.exit(1)` and suggests `throw error` for this.

Reply:
I guess, there is no need to catch at all. Just `main()`. The error message will be printed, together with the stack trace.

Reply:
got it, I will remove the unnecessary throw errors